mlflow/entities/trace_info.py

```diff
@@ -44,7 +44,10 @@ def to_proto(self):
         proto.request_id = self.request_id
         proto.experiment_id = self.experiment_id
         proto.timestamp_ms = self.timestamp_ms
-        proto.execution_time_ms = self.execution_time_ms
+        # NB: Proto setter does not support nullable fields (even with 'optional' keyword),
+        # so we substitute None with 0 for execution_time_ms. This should be not too confusing
+        # as we only put None when starting a trace i.e. the execution time is actually 0.
+        proto.execution_time_ms = self.execution_time_ms or 0
```

Comment:
note: If we don't do this, we'll get `TypeError: None has type NoneType, but expected one of: int`